examples/pxScene2d/src/Makefile

```diff
@@ -414,6 +419,10 @@ PXSCENE_EXTRA_DEFINES=-DPNG_APNG_SUPPORTED
 endif
 endif
 
+ifeq ($(HAVE_RT_REMOTE),1)
+EXTRA_COMPILER_FLAGS:=$(EXTRA_COMPILER_FLAGS) -DUSE_RT_REMOTE 
+LDFLAGS_FULL:=$(LDFLAGS_FULL) -L../../../remote -lrtRemote
```

Comment:
Enabled Service Manager support for libpxscene.so.
Actually I've enabled it to 3 targets only: pxscene, libpxscene.so and libpxscene.a.